content/lizheming/drone-wechat/index.md

```diff
@@ -0,0 +1,161 @@
+---
+date: 2018-01-07T00:00:00+00:00
+title: Wechat
+author: lizheming
+tags: [ notifications, chat ]
+repo: lizheming/drone-wechat
+logo: wechat.svg
+image: lizheming/drone-wechat
+---
+
+The Wechat plugin posts build status messages to your account. The below pipeline configuration demonstrates simple usage:
+
+```diff
+pipline:
+  wechat:
+    image: lizheming/drone-wechat
++   sckey: xxxxxx
+    title: ${DRONE_REPO_NAME}
+    message: >
+      {%if success %}
+        build {{build.number}} succeeded. Good job.
+      {% else %}
+        build {{build.number}} failed. Fix me please.
+      {% endif %}
+```
+
+If you want push notification with your own wechat corp id, you can config like this:
+
+```diff
+pipeline:
+  wechat:
+    image: lizheming/drone-wechat
++   corpid: corpid
++   corp_secret: secret
++   agent_id: 1234567
++   to_user: 111
++   to_party: 112
++   to_tag: ${DRONE_REPO_NAME}
++   msg_url: ${DRONE_BUILD_LINK}
++   safe: 1
++   btn_txt: more
+    title: ${DRONE_REPO_NAME}
+    message: >
+      {%if success %}
+        build {{build.number}} succeeded. Good job.
+      {% else %}
+        build {{build.number}} failed. Fix me please.
+      {% endif %}
+```
+
+# Parameter Reference
+
+corpid
+: The corpid for authorization
+
+corp_secret
+: The corp secret for authorization
+
+agent_id
+: The agent id to send the message
+
+to_party
+: The party ids to send message
+
+to_user
+: The user ids to send the message to
+
+to_tag
+: The tag ids to send the message to
+
+safe
+: encrypt message, default is false
+
+msg_url
+: The link for the text card click
+
+btn_text
+: The text for the button on the card
+
+sckey
+: SCKEY get from [ServerChan](http://sc.ftqq.com)
+
+title
+: Notification title
+
+message
+: Notification body message, support markdown
+
+# Secret Reference
+
+wechat_corpid
+: The corpid for authorization
+
+wechat\_corp\_secret
```

Comment:
`\` in many places. Please remove all.

Reply:
`_corp_` will be  recognized italic style in markdown, so we need add slash to forbidden it. just show difference like this:

wechat_corp_secret  and wechat\_corp\_secret

Reply:
OK, I removed them. VSCode tips me it's an italic style in markdown, but it seems github doesn't recognize it. 

![image](https://user-images.githubusercontent.com/424491/34703103-2323665c-f52c-11e7-9d1f-c41a80dfa004.png)